assets/app/actions/createDraftBranchName.js

```diff
@@ -0,0 +1,5 @@
+import { encodeB64 } from '../util/encoding';
+
+export default path => {
```

Comment:
Yes, the `util` directory is also lazy :( Is there a good way to define some small subset of folders we can group these functions into? It would be nice to come up with something that would be mostly obvious or easily documented so that future utility type functions could be easily added to the right folder.

Something like `services` for API interactions, `transforms` for data transforms, `filters` for things that take a list and reduce the contents. I don't know exactly what the naming conventions should be and I don't want to drift too far into angular land with them.

Does that seem like a reasonable approach?